src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private int buy = 0;
+    private int supply = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
```

Comment:
```suggestion
       String fileData = readFromFile(fromFileName);
       String report = createReport(fileData);
       writeToFile(toFileName, report);
```

 `writeToFile(toFileName, createReport(readFromFile(fromFileName)));` it is really hard to read